src/options/optionsMain.js

```diff
@@ -32,6 +32,8 @@ const DEFAULT_APP_NAME = 'APP';
 function optionsFactory(inpOptions, callback) {
 
     const options = {
+        'app-copyright': inpOptions['app-copyright'],
+        'app-version': inpOptions['app-version'],
```

Comment:
I believe you can camel case the object keys here, like inpOptions.appCopyright as [commander](https://github.com/tj/commander.js/) does that for you